src/components/CarouselDots.js

```diff
@@ -10,6 +10,7 @@ class CarouselDots extends Component {
     value: PropTypes.number,
     onChange: PropTypes.func,
     rtl: PropTypes.bool,
+    className: PropTypes.string,
```

Comment:
could you add it to the README?

Reply:
Will do!

Reply:
@piotr-s-brainhub let me know if that readme works and what else I might need to change?

Reply:
I'll probably need some help on making the build work too....